charts/observability-pipelines-worker/templates/haproxy/deployment.yaml

```diff
@@ -0,0 +1,137 @@
+{{- if .Values.haproxy.enabled }}
```

Comment:
@lukesteensen @binarylogic this may be contentious but I'd recommend removing HAProxy from the chart. @silversupreme has mentioned to me that it has caused some problems in the past and that for testing/initial onboarding a single instance should be suitable.

Reply:
I don't have much context here, but anything that keeps things simpler seems like a good idea to me.